EndpointHandler.php

```diff
@@ -0,0 +1,110 @@
+<?php
+
+require "settings.php";
```

Comment:
Actually, I don't think 'settings' even needs to be included at the top.

---
In reply to: [71211850](https://github.com/AzureAD/active-directory-b2c-wordpress-plugin-openidconnect/pull/3#discussion_r71211850) [](ancestors = 71211850)